phabfive/cli.py

```diff
@@ -115,6 +115,19 @@
     phabfive maniphest comment add <ticket_id> <comment> [options]
     phabfive maniphest show <ticket_id> ([--all] | [--pp]) [options]
     phabfive maniphest create <config-file> [--dry-run] [options]
+    phabfive maniphest search [options]
```

Comment:
Non-optional arguments are not passed as a generic "`[options]`" in docopt. You should use the "`<board_id>`" variant. Read the docopt documentation or look at other code examples in this repository.